HW_12/src/beer.ts

```diff
@@ -0,0 +1,24 @@
+import { Meal } from "./meel";
+import { beerValue, beerBrand} from "./enums"
+
+export interface IBeer {
+    beerValue: beerValue;
+    beerBrand: beerBrand;
+    calculatePrice(): number;

```

Comment:
This method is already declared in the abstract class `Meal`

Reply:
u a right